test/convertStringToBinary.test.js

```diff
@@ -0,0 +1,12 @@
+const { convertStringToBinary } = require("../lib");
+const SUCCESFULL_BINARY_STRING = "01010100 01101000 01101001 01110011 " +
```

Comment:
Just a small typo: it should be "SUCCESSFULL" - make sure to adapt that change in the test as well.